src/main.rs

```diff
@@ -1,3 +1,27 @@
-fn main() {
-    println!("Hello, world!");
+mod nma {
+    pub mod nma;
+}
+
+pub mod u5;
+pub mod u6;
+
+pub mod args;
+pub mod config;
+
+use args::parse_args;
+use config::parse_config;
+use nma::nma::assemble;
+
+fn main() -> Result<(), &'static str> {
+    let cmd_args = parse_args()?;
+    let config = parse_config(&cmd_args).expect("Failed to parse configuration file");
```

Comment:
``.expect()`` immediately kills the program if an error occurs, let's not do that for error-prone filesystem access